test-harness/index.ts

```diff
@@ -17,6 +17,9 @@ describe('cli acceptance tests', () => {
   describe.each(files)('%s file', file => {
     const data = fs.readFileSync(path.join(cwd, file), { encoding: 'utf8' });
     const scenario = parseScenarioFile(data);
+
+    if (scenario.command === void 0) return;
```

Comment:
@P0lip Will this mark the test as 'skipped' in the console? Wouldn't that be the case, how complex would it be to make it happen? That would be pretty handy to detect scenario that are not fully xplat...

Reply:
Nah, it won't. Can make it marked though.